pages/public_cloud/ai_machine_learning/deploy_tuto_15_rock_paper_scissors/guide.en-gb.md

```diff
@@ -0,0 +1,175 @@
+
+
+### Write the Streamlit application
+
+Create a Python file named `app.py` and past the following code:
+
+```python
+from ultralytics import YOLO
+import streamlit as st
+
+#######################################################################################################################
+## 🎯 The aim of this script is to create an Rock/Paper/Scissors application based on a trained model (from YOLOv8). ##
+## 🏞 The uploaded snapshots are stored in /workspace/                                                               ##
+## 🧠 The train model is stored in /workspace/model/rock-paper-scissors/                                             ##
+#######################################################################################################################
+
+# Save uploaded photo
+def save_photo(photo):
+    
+    photoAbsolutePath = '/workspace/' + photo.name
+    
+    with open(photoAbsolutePath,'wb') as f:
+         f.write(photo.getbuffer())
+    
+    return photoAbsolutePath
+
+# main
+if __name__ == '__main__':
+
+    st.write("## Welcome on the 🪨 📄 ✂️ game!")
+    # 🧠 Load the model
+    model = YOLO('/workspace/model/rock-paper-scissors/best.torchscript')
+
+    # 📸 Camera input
+    img_file_buffer = st.camera_input("Take your picture in real time:")
+    if img_file_buffer is not None:
+      photoPath = save_photo(img_file_buffer) 
+
+      # 🔎 Prediction
+      results = model.predict(photoPath, verbose=True, save=True, conf=0.5)
+
+      # 📈 Display results
+      for r in results:
+        for c in r.boxes.cls:
+          st.write(r.names[int(c)])
+```
+
+### Write the requirements.txt file for the application
+
+The `requirements.txt` file will allow us to write all the modules needed to make our application work. This file will be useful when writing the `Dockerfile`.
+
+```console
+ultralytics
+opencv-python-headless
+streamlit
+```
+
+### Write the Dockerfile for the application
+
+Your Dockerfile should start with the `FROM` instruction indicating the parent image to use. In our case we choose to start from a `python:3.8` image:
+
+```dockerfile
+# 🐳 Base image to execute Python application
+FROM python:3.8
+
+# 👱 Userspace for AI Deploy
+WORKDIR /workspace
+# 📚 Libraries for the application
+ADD requirements.txt /workspace
+RUN pip install -r requirements.txt
+RUN apt-get update
+RUN apt-get install ffmpeg libsm6 libxext6  -y
+
+# 🐍 Python application
+ADD app.py /workspace
+
+# 👮 Rootless execution
+RUN chown -R 42420:42420 /workspace
+
+# 🚀 Run the Python application
+CMD [ "streamlit" , "run" , "/workspace/app.py", "--server.address=0.0.0.0" ]
+
+ENV HOME=/workspace
+```
+### Build the Docker image from the Dockerfile
+
+Launch the following command from the **Dockerfile** directory to build your application image:
+
+```bash
+docker build . -f Dockerfile -t <regristry-name>/rock-paper-scissors-app:1.0.0
```

Comment:
Since you do not use docker tag, I think you need to add the shared registry warning and explanations before the docker build command, since it specifies the registry.

And according to the following parts : 
```suggestion
docker build . -f Dockerfile -t <shared-regristry-name>/rock-paper-scissors-app:1.0.0
```